Microsoft.Toolkit.Uwp.UI.Controls/Menu/Menu.xaml

```diff
@@ -47,8 +48,7 @@
                             FontStyle="{TemplateBinding FontStyle}"
                             FontWeight="{TemplateBinding FontWeight}"
                             Foreground="{TemplateBinding Foreground}"
-                            IsTabStop="False"
-                            Padding="{TemplateBinding Padding}">
```

Comment:
Why is this being changed in this PR?

Reply:
No clue !! maybe I was copying and pasting.

Reply:
That was because of Xaml Styler